tests/csm/rest/test_iam_users.py

```diff
@@ -4512,3 +4512,51 @@ def test_39406(self):
             assert_utils.assert_true(resp[0], resp[1])
         self.log.info("[END]Creating IAM users with different tenant")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42270')
+    def test_42270(self):
+        """
+        Test GET IAM user with valid max_entries and marker using Admin login
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Creating 10 IAM users.")
+        users_list = []
+        for count in range(10):
+            resp = self.csm_obj.verify_create_iam_user_rgw(verify_response=True)
+            assert_utils.assert_true(resp[0], resp[1])
```

Comment:
Use assert resp[0], resp[1], as it prints both sides of condition

Reply:
In this case as well it will print resp[1] as well isn't it? Why do we need assert only?